doc/conf.py

```diff
@@ -19,18 +19,23 @@
 ]
 
 source_suffix = ".rst"
-master_doc = "index"
+root_doc = "index"
 
 autoclass_content = "both"
 autodoc_member_order = "bysource"
 default_role = "py:obj"
 
 html_theme = "sphinx_rtd_theme"
+html_theme_options = {
+    "prev_next_buttons_location": "both",
```

Comment:
When I look at the docs artifact from the build (https://amazon-braket-sdk-python--758.org.readthedocs.build/en/758/) I still only see the next button.

Reply:
Is this only occurring on the landing page? If so, that's expected. Navigating using the next button, I can then see the previous button appear.

Reply:
Oh, of course, that makes perfect sense! 🤦‍♀️